File: 프로그래머스/unrated/120908. 문자열안에 문자열/문자열안에 문자열.py
def solution(str1, str2):
    flag = 1
    for i in range(len(str1)):
        if str2[0] == str1[i]:
            j=0
            flag = 0
            while j+i<len(str1) and j < len(str2):
                if str1[j+i] != str2[j]:
                    flag = 1
                    break
                j+=1
        if flag == 0 and j == len(str2):
            return 1
    return 2

File: 프로그래머스/0/120908. 문자열안에 문자열/문자열안에 문자열.java
class Solution {
    public int solution(String str1, String str2) {
        String[] sa1 = str1.split("");
        String[] sa2 = str2.split("");
        int lenSa1 = sa1.length, lenSa2 = sa2.length;
        
        
        for(int i=0;i<lenSa1 - lenSa2 + 1;i++){
            if(sa1[i].equals(sa2[0])){
                int cnt = 0;
                for(int j=0;j<lenSa2;j++){
                    if(sa1[i+j].equals(sa2[j])) cnt++;
                }
                if(cnt == lenSa2)
                    return 1;
            }
        }
        
        return 2;
    }
}